src/main/java/ch/njol/skript/expressions/ExprWorldBorderCenter.java

```diff
@@ -0,0 +1,67 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Location;
+import org.bukkit.WorldBorder;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Center of World Border")
+@Description("The center of a world border.")
+@Examples("set border center of {_worldborder} to location(10, 0, 20)")
+@Since("INSERT VERSION")
+public class ExprWorldBorderCenter extends SimplePropertyExpression<WorldBorder, Location> {
+
+	static {
+		register(ExprWorldBorderCenter.class, Location.class, "[world[ ]]border (center|middle)", "worldborders");
+	}
+
+	@Override
+	@Nullable
+	public Location convert(WorldBorder worldBorder) {
+		return worldBorder.getCenter();
+	}
+
+	@Override
+	@Nullable
+	public Class<?>[] acceptChange(ChangeMode mode) {
+		switch (mode) {
+			case SET:
+			case RESET:
+				return CollectionUtils.array(Location.class);
+		}
```

Comment:
I don't believe reset should be an option as there is no real default location of a border, opinions welcomed

Reply:
When a new worldborder is created it defaults to 0,0 so I feel like that's a valid location to reset to

Reply:
Hmm I see do we believe there's any use in having a reset tho? since iirc center refers to where the border is located in the world and this feels like it'll causes unintended issues